chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -764,12 +763,7 @@ impl PeerManagerActor {
                     NetworkResponses::RouteNotFound
                 }
             }
-            NetworkRequests::StateRequestHeader { shard_id, sync_hash, target } => {
-                let peer_id = match target {
-                    AccountOrPeerIdOrHash::AccountId(_) => return NetworkResponses::RouteNotFound,
-                    AccountOrPeerIdOrHash::PeerId(peer_id) => peer_id,
-                    AccountOrPeerIdOrHash::Hash(_) => return NetworkResponses::RouteNotFound,
```

Comment:
I can't find any places where `AccountOrPeerIdOrHash::Hash(_)` have been constructed. Was that a dead enum variant already?

Reply:
It seems so to me too.